### ISSUE TITLE
[A11y Audit]: Product Pages

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5501]
### Pages:
- `/code-search`
- `/batch-changes`
- `/code-insights`
- **note**: `/code-intelligence` is hosted on Docs
- `/code-intelligence-platform`
- `/sourcegraph-4`

### User Journey:
- Navigate to the above pages.
- All links can be clicked through.
- All carousels can be experienced.
- "Install Sourcegraph locally" script can be copied to clipboard. (on `/code-search`)
- All imagery is accessible.
- Any video can pause or play, and has CC.
- Any form can be completed.
- Any tooltip is accessible through keyboard or screen reader.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

Video example [here](https://www.loom.com/share/21a5d671fc2a44b6be9f61335389c456).

1. Navigate user journey using **only** the keyboard
2. Navigate user journey **without** looking at your screen, using the screen reader.
3. Navigate user journey using a viewport with a **width of 320px**.
4. Run an [ARC Toolkit test](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-arc-toolkit) on the webpage for any changing state.
5. If the issue is trivial & you have time, feel free to address the bug now. Otherwise, raise any larger bugs using the [accessibility issue template](https://github.com/sourcegraph/about/issues/new?assignees=&labels=accessibility%2Cwcag%2F2.1%2Cwcag%2F2.1%2Ffixing&template=accessibility_issue.yaml&title=%5BAccessibility%5D%3A+).

**Note:** [The Front End Platform team](https://join.slack.com/share/enQtMzY1OTY3MDkxNTAwOS03ZmFlNzgwNWJhOGU3NjU4MDY1NzVkOTdiNjQwMWQyNjJmNjBkNzk1OWQ5ZDIxOTNjNmRhOWIyMzUwMzhmZjNh) is planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!
